.github/workflows/triage-prs-to-board.yml

```diff
@@ -7,12 +7,14 @@ on:
 
 jobs:
   pr-triage:
-    uses: 'yuvipanda/pr-triage-board-bot/.github/workflows/reusable-pr-triage.yml@main'
-    with:
-      organization: 'geojupyter'
-      project-number: '3'
-      gh-app-id: '1943749'
-      gh-installation-id: '85628873'
-      repositories: 'jupytergis'
-    secrets:
-      gh-app-private-key: '${{ secrets.GH_APP_PRIVATE_KEY }}'
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: 'Run PR Triage Bot'
+        uses: 'yuvipanda/pr-triage-board-bot@c350212c2bba8a47cc2b07e7ce40fd6748b6beef'
```

Comment:
@jasongrout time to hash-pin or no? :)

Reply:
oh nice, the hash ends in "beef"!

Reply:
I noticed that too :laughing: I love hashes with fun patterns in them :laughing: